test/rejection-handler.test.js

```diff
@@ -70,21 +70,19 @@ describe('UnhandledRejectionHandler', function () {
 
   it('.handle()', function (done) {
     var existing = helpers.clearRejections();
-    var writeable = new stream.Writable({
-      objectMode: true,
-      write: function (info) {
-        assume(info).is.an('object');
-        assume(info.error).is.an('error');
-        assume(info.error.message).equals('wtf this rejection');
-        assume(info.message).includes('unhandledRejection: wtf this rejection');
-        assume(info.stack).is.a('string');
-        assume(info.process).is.an('object');
-        assume(info.os).is.an('object');
-        assume(info.trace).is.an('array');
-
-        existing.restore();
-        done();
-      }
+    var writeable = helpers.writeable(function (info) {
+      console.log('in writeable', info);
```

Comment:
This is never reached. It's like it's not being attached the way the test expects.